cmd/gaia/testnets/README.md

```diff
@@ -165,64 +109,64 @@ Check the everything is running smoothly:
 gaiacli status
 ```
 
-## Generate keys
+View the status of the network with the [Cosmos Explorer](https://explorecosmos.network).Once your full node syncs up to the current block height, you should see it appear on the [list of full nodes](https://explorecosmos.network/validators).
+
+## Generate Keys
 
 You'll need a private and public key pair \(a.k.a. `sk, pk` respectively\) to be able to receive funds, send txs, bond tx, etc.
 
-To generate your a new key \(default _ed25519_ elliptic curve\):
+To generate a new key \(default _ed25519_ elliptic curve\):
 
 ```
 gaiacli keys add <your_key_name>
 ```
 
-Next, you will have to enter a passphrase for your key twice. Save the _seed_ _phrase_ in a safe place in case you forget the password.
+Next, you will have to create a passphrase. Save the _seed_ _phrase_ in a safe place in case you forget the password.
 
-Now if you check your private keys you will see the `<your_key_name>` key among them:
+Now if you check your private keys you will see `<your_key_name>` among them:
 
 ```
 gaiacli keys show <your_key_name>
 ```
 
-You can see all your other available keys by typing:
+You can see all your available keys by typing:
 
 ```
 gaiacli keys list
 ```
 
-The validator pubkey from your node should be the same as the one printed with the command:
+The validator pubkey for your node is:
 
 ```
 gaiad tendermint show_validator
 ```
 
-Finally, save your address and pubkey into a variable to use them afterwards.
+Lastly, save your address and pubkey to environment variables for later use.
 
 ```
 MYADDR=<your_newly_generated_address>
 MYPUBKEY=<your_newly_generated_public_key>
 ```
 
-**IMPORTANT:** We strongly recommend to **NOT** use the same passphrase for your different keys. The Tendermint team and the Interchain Foundation will not be responsible for the lost of funds.
+**IMPORTANT:** We strongly recommend **NOT** using the same passphrase for all your keys. The Tendermint team and the Interchain Foundation will not be responsible for the loss of funds.
 
-### Get coins
+### Get Tokens
 
-The best way to get coins at the moment is to ask in Riot chat. We plan to have a reliable faucet in future testnets.
+The best way to get tokens is from the [Cosmos Testnet Faucet](https://faucetcosmos.network). If the faucet is not working for you, try asking [#cosmos-validators](https://riot.im/app/#/room/#cosmos-validators:matrix.org).
```

Comment:
May also want to add, "if you were a validator on previous testnets, you can submit a PR to add yourself to the next genesis file, no guarantees that it will get accepted though. We're currently working on tooling to make this more easy."

Reply:
This section is being added by Zaki's PR here: https://github.com/cosmos/cosmos-sdk/pull/1211

Reply:
Cool! Didn't see that